_migration-assistant/deploying-migration-assistant/getting-started-data-migration.md

```diff
@@ -145,10 +145,16 @@ Use the following steps to configure and deploy RFS, deploy Migration Assistant,
             }
         },
         "reindexFromSnapshotExtraArgs": "<RFS PARAMETERS (see below)>",
+        "reindexFromSnapshotServiceEnabled": true,
```

Comment:
This flag is enabled by default, lets keep this context definition as simple as possible and remove this value.